legend-engine-test-runner-service/src/main/java/org/finos/legend/engine/testable/service/extension/ServiceTestRunner.java

```diff
@@ -347,6 +353,22 @@ private org.finos.legend.engine.protocol.pure.v1.model.test.result.TestResult ex
 
                 testResult = testFailed;
                 testResult.atomicTestId = atomicTestId;
+
+                testFailed.assertStatuses.forEach(assertion ->
+                {
+                    if (assertion instanceof EqualToJsonAssertFail)
+                    {
+                       span.log("ASSERTION: " + assertion.id + " FAILED : {\"expected\": " + ((EqualToJsonAssertFail)assertion).expected + ", \"actual\": " + ((EqualToJsonAssertFail)assertion).actual + "}");
```

Comment:
we should log these separately. 
Span.log(assert failed..)
Span.log(expected:..)
Span.log(actual:..) 
instead of building a json